src/components/Skeleton/Skeleton.module.scss

```diff
@@ -1,3 +1,17 @@
+@import "SkeletonVariables";
+
 .fullWidth {
   width: 100%;
 }
+
+:global(.monday-style-skeleton):not(.fullWidth) {
```

Comment:
Probably better to find some other way here, as we want to remove old classnames soon

Reply:
at the moment we will remove old class name we will replace :global(.monday-style-skeleton) with the new skeleton css class name